config/examples/Creality/Ender-5 Plus/BigTreeTech SKR 3/README.md

```diff
@@ -5,10 +5,12 @@
 >
 > |  LCD -> SKR 3   |
 > |-----------------|
-> | `TX2` -> `PA10` |
+> | `+5V` -> `+5V`  |
+> | `RX4` -> ``     |
 > | `RX2` -> `PA9`  |
+> | `TX2` -> `PA10` |
+> | `TX4` -> ``     |
```

Comment:
If you preview the README, double back ticks show up as such. If these are not connected, `NC`  or an asterisk with “* Not connected” text below the connector would be clearer.